src/vm/arm64/stubs.cpp

```diff
@@ -798,6 +798,24 @@ void ResumableFrame::UpdateRegDisplay(const PREGDISPLAY pRD)
     pRD->ControlPC = m_Regs->Pc;
     pRD->SP = m_Regs->Sp;
 
+    pRD->volatileCurrContextPointers.X[0] = &m_Regs->X0;
+    pRD->volatileCurrContextPointers.X[1] = &m_Regs->X1;
```

Comment:
You should update the definition of the CONTEXT structure in the PAL to what is in Windows headers instead, and undo this change.